### PR TITLE
Update django version requirement

### PR DESCRIPTION
**Pull request by webbyfox**
- [x] Changelog entry added.
 - [x] Upgraded any vulnerable dependencies.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
     },
     include_package_data=True,
     install_requires=[
-        'django>=1.11,<=3.1',
+        'django>=1.11,<=2.2.19',
```
Comment (depsiatwal) [inline setup.py:25]: correct me if i'm wrong but doesn't this just restrict client lib on what version is compatible ?
so still we can install 1.11 to 3.1  what advantage is it to restrict it to 2.2.19

Comment (stevejalim) [reply setup.py:25]: Ah yeah, you're right - i think i we just change the upper limit to the latest secure 3.1.x -> 3.1.7

Comment (stevejalim) [inline setup.py:25]:
```suggestion
        'django>=1.11,<=3.1.7,
```

@depsiatwal  @webbyfox  How about this instead?

Comment (stevejalim) [reply setup.py:25]:
```suggestion
        'django>=1.11,<=3.1.7',
```

How about this instead @webbyfox  @depsiatwal

**Review by webbyfox — COMMENTED** [review]
..

File: setup.py (hunk 1 above)
Comment (webbyfox) [reply setup.py:25]: I guess this is just to make the Github alert bot happy. the actual installation will not force to go higher version if there is already Django exist. happy with this.

**Review by stevejalim — APPROVED** [review]
LGTM

**Pull request merged**